src/mono/mono/component/event_pipe-stub.c

```diff
@@ -4,6 +4,7 @@
 
 #include <config.h>
 #include "mono/component/event_pipe.h"
+#include "mono/component/event_pipe-wasm.h"
```

Comment:
The header itself has the ifdefs. Could do it either way, but this way the ifdefs are in one place instead of in every .c file. but I'm not opposed to changing it